src/jaxoplanet/experimental/starry/visualization.py

```diff
@@ -1,12 +1,12 @@
 import numpy as np
 
-from jaxoplanet.experimental.starry.maps import Map
+from jaxoplanet.experimental.starry.surface import Surface
 from jaxoplanet.experimental.starry.utils import graticule
 from jaxoplanet.experimental.starry.ylm import Ylm
 
 
 def show_map(
```

Comment:
Should this be renamed to `show_surface`?

Reply:
Absolutely! I will refactor